userenvs/alma8/userenv.json

```diff
@@ -0,0 +1,14 @@
+  "userenv": {
+    "name": "alma8",
```

Comment:
just curious if alma is better than rocky for replacing centos

Reply:
I did a little research on this and decided it was likely a wash, but for some reason Alma seemed like a more comfortable change to me.